js/utility.js

```diff
@@ -21,14 +21,11 @@ const Utils = {
     return out;
   },
 
-  getCurrentURL(router=false) {
+  getCurrentURL() {
```

Comment:
`getCurrentURL()` is never used with an `argument` passed in. I guess this is just some old code that we forgot to update.